### PR TITLE
Demo app with secrets

### PR DESCRIPTION
**Pull request by wzieba**
### Description

This PR adds support for `mobile-secrets` in order to provide OAuth secrets for the demo-app prototype build.

### Testing Steps

Install the app from the comment below and verify that Quick Editor works as expected.

### CODE REVIEW

File: demo-app/build.gradle.kts [hunk 1]
```diff
@@ -10,13 +10,13 @@ plugins {
...
-fun localProperties(): Properties {
-    val localProperties = Properties()
-    val localPropertiesFile = rootProject.file("local.properties")
-    if (localPropertiesFile.exists()) {
-        localProperties.load(FileInputStream(localPropertiesFile))
-    }
-    return localProperties
+fun secretProperties(): Properties {
+    val properties = Properties()
+    rootProject.file("secrets.properties")
```
Comment (wzieba) [inline demo-app/build.gradle.kts:15]: After merge, the team will have to move their current secrets from `local.properties` to `secrets.properties` or run `bundle exec fastlane run configure_apply` to use the shared secrets.

**Review by AdamGrzybkowski — APPROVED** [review]
Tested the build from the PR and it worked, thanks! 

One question: What steps would we have to take to add new properties (if needed)?

File: demo-app/build.gradle.kts (hunk 1 above)
Comment (AdamGrzybkowski) [reply demo-app/build.gradle.kts:15]: When I try to run this command I get:
>[!] Decryption key could not be found

**Comment by wzieba** [comment]
> One question: What steps would we have to take to add new properties (if needed)?

Good question! With review comment I see you don't have `.mobile-secrets` setup locally. Please configure this (check out "mobile secrets" page on the field guide) and then just edit `android/Gravatar-SDK-Android/secrets.properties` file and push the change.

After 2m, in this repository you have to run `bundle exec fastlane configure_update` and push the updated `.enc` file. It's all explained there with more details in the guide 👍

**Pull request merged**